Modules/Core/SpatialObjects/test/itkSpatialObjectToImageFilterTest.cxx

```diff
@@ -51,31 +52,43 @@ itkSpatialObjectToImageFilterTest(int, char *[])
 
   using SpatialObjectToImageFilterType = itk::SpatialObjectToImageFilter<EllipseType, ImageType>;
   SpatialObjectToImageFilterType::Pointer imageFilter = SpatialObjectToImageFilterType::New();
+
+  ITK_EXERCISE_BASIC_OBJECT_METHODS(imageFilter, SpatialObjectToImageFilter, ImageSource);
+
+
   imageFilter->SetInput(ellipse);
-  imageFilter->SetInsideValue(2);
-  imageFilter->GetInsideValue();
+
+  SpatialObjectToImageFilterType::ValueType insideValue = 2;
+  imageFilter->SetInsideValue(insideValue);
+  ITK_TEST_SET_GET_VALUE(insideValue, imageFilter->GetInsideValue());
+
+  SpatialObjectToImageFilterType::ValueType outsideValue = 0;
   imageFilter->SetOutsideValue(0);
-  imageFilter->GetOutsideValue();
-  imageFilter->SetChildrenDepth(1);
-  imageFilter->GetChildrenDepth();
+  ITK_TEST_SET_GET_VALUE(outsideValue, imageFilter->GetOutsideValue());
+
+  unsigned int childrenDepth = 1;
+  imageFilter->SetChildrenDepth(childrenDepth);
+  ITK_TEST_SET_GET_VALUE(childrenDepth, imageFilter->GetChildrenDepth());
+
   ImageType::SizeType size;
   size[0] = 50;
   size[1] = 50;
   imageFilter->SetSize(size);
+  ITK_TEST_SET_GET_OBJECT(size, imageFilter->GetSize());
```

Comment:
Modules/Core/SpatialObjects/test/itkSpatialObjectToImageFilterTest.cxx:77:3: error: 'ITK_TEST_SET_GET_OBJECT' was not declared in this scope

Reply:
Awesome catch @dzenanz 👌 . Thanks. Fixed in 4c20457.